src/openforms/js/components/form/edit/tabs.js

```diff
@@ -84,6 +84,31 @@ const TEXT_BASIC = {
     ]
 };
 
+
+const FIELDSET_BASIC = {
```

Comment:
please move this to the fieldset component module, it's not used in any other components.